features/admin/pod.feature

```diff
@@ -10,6 +10,7 @@ Feature: pod related features
   @network-ovnkubernetes @network-openshiftsdn
   @proxy @noproxy
   @heterogeneous @arm64 @amd64
+  @ppc64le
```

Comment:
Hi, is it possible to add them in the same row as the other architectures?

Reply:
@aleskandro Thanks for reviewing. Yes, I've added them in the same line as other architectures.